dojo/settings/settings.dist.py

```diff
@@ -276,6 +281,12 @@ def generate_url(scheme, double_slashes, user, password, host, port, path):
 SOCIAL_AUTH_OKTA_OAUTH2_SECRET = env('DD_SOCIAL_AUTH_OKTA_OAUTH2_SECRET')
 SOCIAL_AUTH_OKTA_OAUTH2_API_URL = env('DD_SOCIAL_AUTH_OKTA_OAUTH2_API_URL')
 
+AZUREAD_TENANT_OAUTH2_ENABLED = True
```

Comment:
Any type of OAuth support is disabled by default.

Reply:
@dougmorato This should be changed to env('DD_AZUREAD_TENANT_OAUTH2_ENABLED')

Reply:
Yep, will do and also include the other integrations (okta, google) on the env_settings re-write